client/components/signup-form/docs/example.jsx

```diff
@@ -0,0 +1,28 @@
+/** @format */
```

Comment:
I think this file is being moved to /blocks in another PR

Reply:
Thanks, I'll have a look at that.